entrypoint.sh

```diff
@@ -77,7 +77,7 @@ if [ -z "$ORYX_DISABLE_TELEMETRY" ] || [ "$ORYX_DISABLE_TELEMETRY" == "false" ];
     export GITHUB_ACTIONS_BUILD_IMAGE_PULL_END_TIME=$endTime
 fi
 
-export ORYX_SDK_STORAGE_BASE_URL="https://oryxsdk-cdn.azureedge.net"
+export ORYX_SDK_STORAGE_BASE_URL="https://oryx-cdn.microsoft.io"
```

Comment:
This export is no longer required since it is part of the image's environment variable list

Reply:
BTW as per today's discussion we do not want to change this action from `v2-beta` to `v2` yet, so for now we can merge this to `v2-beta` itself.